packages/compass-e2e-tests/helpers/commands/run-find-operation.js

```diff
@@ -72,34 +77,34 @@ async function collapseOptions(client, tabName) {
   // results as if you ran the whole suite. If we ever do want to test that all
   // the options you had set before you collapsed the options are still in
   // effect then we can make this behaviour opt-out through an option.
-  await setProject(client, tabName, '');
-  await setSort(client, tabName, '');
-  await setMaxTimeMS(client, tabName, '');
-  await setCollation(client, tabName, '');
-  await setSkip(client, tabName, '');
-  await setLimit(client, tabName, '');
-
-  await client.clickVisible(Selectors.queryBarOptionsToggle(tabName));
-  await waitUntilCollapsed(client, tabName);
+  await setProject(browser, tabName, '');
+  await setSort(browser, tabName, '');
+  await setMaxTimeMS(browser, tabName, '');
+  await setCollation(browser, tabName, '');
+  await setSkip(browser, tabName, '');
```

Comment:
This is the line that suddenly started giving me trouble: It would clear the skip field from '5' to '', then the moment the field loses focus compass sets it back to '5'. Eventually I gave up for now.